.github/workflows/build.yml

```diff
@@ -135,7 +135,7 @@ jobs:
         run: bundle exec rake db:setup
 
       - name: ${{ matrix.tests }} tests with feature flags ${{ matrix.feature-flags }}
-        run: bundle exec --verbose rspec --exclude-pattern ${EXCLUDE_PATTERN} --pattern ${INCLUDE_PATTERN} --format documentation
+        run: apk add chromium chromium-chromedriver && bundle exec --verbose rspec --exclude-pattern ${EXCLUDE_PATTERN} --pattern ${INCLUDE_PATTERN} --format documentation
```

Comment:
Can we move adding those to a different step? Also Im wondering around this slowing down our builds, Github actions has caching capabilities, is it possible to extend them for this step

Reply:
Good point, as discussed I'm looking into github actions caching

Reply:
I've moved it to a different step.

Not sure about the caching though - I was looking to borrow from other projects, but they don't cache chromedriver (e.g. https://github.com/DFE-Digital/register-trainee-teachers/blob/master/.github/workflows/build.yml#L59) 

It takes 8s to do the step, maybe we can revisit caching in another PR?

Reply:
sure, as there is scope for caching multiple steps